app/services/requirements/topic_checker.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Requirements
+  class TopicChecker
+    TITLE_MAX_LENGTH = 150
```

Comment:
I don't think you meant for this to be here.

Reply:
Thanks :-).